lib/drs-server/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:3.6-alpine
+
+LABEL Maintainer="CanDIG Project"
+
+USER root
+
+RUN apk update
+
+RUN apk add \
+	autoconf \
+	automake \
+	make \
+	gcc \
+	perl \
+	bash \
+	build-base \
+	musl-dev \
+	zlib-dev \
+	bzip2-dev \
+	xz-dev \
+	libcurl \
+	curl \
+	curl-dev \
+	yaml-dev \
+	libressl-dev \
+	git \
+	postgresql-dev \
+	libffi-dev
+
+RUN mkdir /app
+WORKDIR /app
+
+RUN git clone --branch minio-backend https://github.com/c3g/chord_drs.git
```

Comment:
I've since merged that branch into master, so you can get rid of that --branch param.